lakeview-sync-tool/src/main/java/ai/onehouse/lakeview/sync/LakeviewSyncConfigHolder.java

```diff
@@ -0,0 +1,206 @@
+package ai.onehouse.lakeview.sync;
+
+import ai.onehouse.lakeview.sync.utilities.IdentitySplitter;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParametersDelegate;
+import org.apache.hudi.common.config.ConfigProperty;
+import org.apache.hudi.common.config.TypedProperties;
+import org.apache.hudi.sync.common.HoodieSyncConfig;
+
+import java.util.List;
+
+import static org.apache.hudi.common.config.HoodieCommonConfig.BASE_PATH;
+
+public class LakeviewSyncConfigHolder {
```

Comment:
nit (feel free to ignore): LakeviewSyncConfigStore instead?

Reply:
this is as per the OSS convention